data/deep missions.txt

```diff
@@ -864,39 +881,43 @@ mission "Deep: Remnant Surveillance"
+			`Instead of sending the Inhibitor Cannon to Ivan's lab, it is transfered to a Deep Security ship which immediately departs. When you meet Ivan at his lab, he explains that the Deep was especially interested in the Remnant technology after reading his previous reports, and that his team would need to study it in a "more secure location."`
 			branch friendly
 				"reputation: Remnant" >= 1
-			`	Ivan hands you 235,500 credits. "Thank you for all the help, Captain <last>. You've done a lot for humanity these past few weeks that you have been helping us. It may take us months to fully understand the technology you have brought us, but rest assured that that once we do understand it the galaxy will be a better place. I may be seeing you later, Captain. Stay safe."`
+			`	Ivan hands you 235,500 credits. "Thank you for all the help, Captain <last>. You've done a lot for humanity in these past few weeks. It may take us months to fully understand the technology you have brought us, but rest assured that that once we do understand it, the galaxy will be a better place. I'll see you later, Captain. Stay safe."`
 				decline
 				
 			label friendly
-			`	Ivan hands you 235,500 credits. "Thank you for all the help, Captain <last>, but I have one last thing to ask of you." Ivan reaches into his pocket and pulls out six objects that you recognize as sensor cubes. "I need you to place two of these on each of the Remnant worlds near their spaceports. This entire time we have been looking at outfits, but we have very little information on the capabilities of Remnant ships. These cubes will allow us to get that information. I'll pay you <payment> out of my own pocket if you do this for me, Captain."`
+			`	Ivan hands you 235,500 credits. "Thank you for all the help, Captain <last>, but I have one last thing to ask." Ivan reaches into his pocket and retrieves six small, black cubes. "Would you bring two of these to each of the Remnant worlds? We've been learning a great deal from the study of the Remnant outfits, but aside from your scanning logs we have very little information on the Remnant ships. If you were to place these near their spaceports, it would be most helpful."`
 			choice
 				`	"You can count on me, Ivan."`
 				`	"Sorry, but this doesn't seem like the right thing to do. You'll need to find someone else to do this for you."`
 					decline
 			
-			`	"Thank you, Captain. Return here once you have finish and I'll hand you the credits."`
+			`	"Thank you, Captain. Return here once you have finished, and I'll ensure you are well compensated for the journey."`
+			`	As you collect the featureless cubes from Ivan, you can't help but notice that they are surprisingly heavy as well.`
```

Comment:
is the player explicitly told these are surveilling cubes by this point? My goal was to simply make it clear that these are the same cubes.